.github/workflows/ci.yml

```
@@ -293,3 +293,14 @@ jobs:
           - typo3-version: "^12.4"
             php-version: "8.3"
             composer-dependencies: highest
+  documentation:
+    name: Documentation
+    runs-on: ubuntu-latest
```

Please use a fixed version here so we won't fail when GitHub updates the `ubuntu-latest` version:

```suggestion
    runs-on: ubuntu-22.04
```